src/app/globals.css

```
@@ -1,6 +1,94 @@
+  font-family: "Pretendard Variable";
+  src: url("/assets/fonts/PretendardVariable.woff2") format("woff2");
+  font-weight: 400 800; /* 400~800까지 가변 */
+  font-display: swap; /* 렌더링 최적화 추가 (웹폰트 로딩 시 가시성을 유지) */
+  font-style: normal;
+}
+
 :root {
+  font-family:
+    "Pretendard Variable", "Apple SD Gothic Neo", "Malgun Gothic", sans-serif;
+
+  font-size: 16px; /* 폰트 base 설정 */
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)